wikipendium/wiki/views.py

```diff
@@ -68,6 +73,45 @@ def cachable_article(request, articleContent, lang=lang):
     return cachable_article(request, articleContent, lang=lang)
 
 
+@cache_page_per_user
+def tag(request, tag):
+
+    articles_with_tag = Article.objects.filter(tags__name__in=[tag])
+    all_newest_contents = Article.get_all_newest_contents_all_languages()
+    filtered_contents = filter(lambda ac: ac.article in articles_with_tag,
+                               all_newest_contents)
+    if len(filtered_contents) == 0:
+        raise Http404
+    if len(filtered_contents) == 1:
+        return HttpResponseRedirect(filtered_contents[0].get_absolute_url())
+    trie = [{
```

Comment:
I thought we had managed to remove most old references to the non-trie, but it seems like some of them still remain too:
https://github.com/stianjensen/wikipendium.no/search?q=trie&type=Code&utf8=✓